lib/rules/story-exports.ts

```diff
@@ -83,14 +86,19 @@ export = createStorybookRule({
           return
         }
 
+        const firstNonImportStatement = program.body.find((n) => !isImportDeclaration(n))
+        const node = firstNonImportStatement || program.body[0] || program
+
         // @TODO: bring apply this autofix with CSF3 release
         // const fix = (fixer) => fixer.insertTextAfter(node, `\n\nexport const Default = {}`)
 
-        context.report({
+        const report = {
           node,
           messageId: 'shouldHaveStoryExport',
           // fix,
-        })
+        } as const
```

Comment:
why as const?

Reply:
I'm foreshadowing the changes regarding typescript improvements from the other open PR